src/github.ts

```diff
@@ -547,6 +582,59 @@ export class GitHub {
     return tags;
   }
 
+  async findMergedPullRequests(number?: number): Promise<MergedGitHubPR[]> {
+    const targetBranch = await this.getDefaultBranch();
+    const response = await this.graphqlRequest({
+      query: `query findMergedPullRequests($owner: String!, $repo: String!, $num: Int!) {
+          repository(owner: $owner, name: $repo) {
+            ref(qualifiedName: $targetBranch) {
+              target {
+                ... on Commit {
+                  history(first: $num) {
+                    nodes {
+                      oid
+                      associatedPullRequests(last: 1) {
+                        nodes {
+                          number
+                          body
+                          title
+                          baseRefName
+                          headRefName
+                          labels(first: 10) {
+                            nodes {
+                              name
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }`,
+      owner: this.owner,
+      repo: this.repo,
+      branch: targetBranch,
+      num: number ?? 10,
```

Comment:
👍 I think perhaps this doesn't need to grab as many merged release PRs, because it's going directly to a well known branch name? or perhaps I'm missing something.